java/flight/flight-sql-jdbc-driver/src/test/java/org/apache/arrow/driver/jdbc/ITDriverJarValidation.java

```diff
@@ -70,7 +70,7 @@ private static JarFile getJdbcJarFile() throws IOException {
 
     assertNotNull(driverClassURL, "Driver jar was not detected in the classpath");
     assertEquals(
-        "Driver jar was not detected in the classpath", "jar", driverClassURL.getProtocol());
+        "jar", driverClassURL.getProtocol(), "Driver jar was not detected in the classpath");
```

Comment:
Both are strings, so the IDE won't complaint in the first place, but how did we miss this in the CIs?

Reply:
Also, I'm curious about that too. 

Actually, I have executed multiple `mvn clean install` commands to verify the changes. I never saw that message before merging the PR (to deal with the Flight Module) into the main branch. 🥲

Perhaps, I suspect that this workflow might be cached, similar to how the JNI workflow failed.

Reply:
Probably... It is fine, let's move forward. Thanks for quickly working on this fix.